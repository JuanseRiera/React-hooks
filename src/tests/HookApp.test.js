import { shallow } from "enzyme";
import { HookApp } from "../HookApp";

describe("Pruebas del modulo hookapp", () => {
  test("Debe cargar el componente correctamente", () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
