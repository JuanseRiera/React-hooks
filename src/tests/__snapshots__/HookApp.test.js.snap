// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del modulo hookapp Debe cargar el componente correctamente 1`] = `
<Fragment>
  <h1>
    Hola mundo
  </h1>
</Fragment>
`;
